src/main/scala/com/codacy/client/bitbucket/v2/service/RepositoryServices.scala

```diff
@@ -19,20 +21,32 @@ class RepositoryServices(client: BitbucketClient) {
       userRole: Option[Role] = None
   ): RequestResponse[Seq[Repository]] = {
     val encodedOwner = URLEncoder.encode(ownerInfo.value, "UTF-8")
-    val baseUrl = s"https://bitbucket.org/api/2.0/repositories/${encodedOwner}"
+    val baseUrl = s"$BASE/${encodedOwner}"
     val role = userRole.fold("")(role => s"role=${role.value}")
     val length = pageLength.fold("")(pagelen => s"pagelen=$pagelen")
     val url = joinQueryParameters(baseUrl, role, length)
     client.executePaginated(Request(url, classOf[Seq[Repository]]))
   }
 
+  /**
+    * Retrieve the repository matching the supplied owner and repositorySlug.
+    *
+    * @param workspace The workspace ID (slug) or the workspace UUID surrounded by curly-braces.
+    * @param repositorySlug The repository slug or the UUID of the repository surrounded by curly-braces
+    * @return A [[RequestResponse]] with the repository data
+    */
+  def getRepository(workspace: String, repositorySlug: String): RequestResponse[Repository] = {
+    val encodedOwner = URLEncoder.encode(workspace, "UTF-8")
```

Comment:
Why do we rename workspace to encodedOwner rather than encodedWorkspace?

Reply:
Because I copy pasted this from other place. Will change it. 👍